CONTRIBUTING.md

```diff
@@ -3,18 +3,21 @@
 Thank you for your interest in contributing to Openverse! This document is
 a set of guidelines to help you contribute to this project.
 
+Please refer to [CONTRIBUTING.md](https://github.com/WordPress/openverse/blob/0b5e24bd325da7b75db9484a268d7c8d197ae920/CONTRIBUTING.md) for a
```

Comment:
I think ideally we'd always link to the latest version here, so that any updates to the upstream contributing doc don't require an update to this link afterwards!
```suggestion
Please refer to [CONTRIBUTING.md](https://github.com/WordPress/openverse/blob/main/CONTRIBUTING.md) for a
```